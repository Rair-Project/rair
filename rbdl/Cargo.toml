[package]
name = "rbdl"
version = "0.1.0"
authors = ["oddcoder <ahmedsoliman@oddcoder.com>"]
edition = "2018"
autotests = false

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
[lib]
name = "rbdl"
proc-macro = true

[dependencies]
syn = {version = "1.0", features=["extra-traits", "full"]}
quote = "1.0"
[[test]]
name = "rbdl_testing"
path = "tests/run_tests.rs"

[dev-dependencies]
trybuild = "1.0"