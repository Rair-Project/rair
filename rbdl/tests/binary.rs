use rbdl::rbdl_file;

rbdl_file!("tests/rbdl/binary.rbdl");

fn main() {

}