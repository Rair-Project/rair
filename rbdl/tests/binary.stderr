error: proc macro panicked
 --> $DIR/binary.rs:3:1
  |
3 | rbdl_file!("tests/rbdl/binary.rbdl");
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: tests/rbdl/binary.rbdl: stream did not contain valid UTF-8
